test/pass/simple-repo.cmake

```diff
@@ -0,0 +1,21 @@
+################################################################################
+# Copyright (C) 2017 Advanced Micro Devices, Inc.
+################################################################################
+
+find_program(GIT NAMES git)
+
+# Try to test without git
+file(MAKE_DIRECTORY ${TMP_DIR}/repo)
+execute_process(COMMAND ${GIT} describe --dirty --long --always WORKING_DIRECTORY ${TMP_DIR}/repo OUTPUT_VARIABLE GIT_TAG RESULT_VARIABLE RESULT)
+if(NOT ${RESULT} EQUAL 0)
+    set(GIT_TAG 0)
+endif()
+write_version_cmake(${TMP_DIR}/repo 1.0 "
+    test_expect_eq(\${PROJECT_GIT_TAG_HASH} ${GIT_TAG})
```

Comment:
This should check the project version. Major should be 1, minor should be 0, patch should be 0, and tweak should be git hash.

Reply:
done. Please review the latest changes.